### DISCUSSION TITLE
Links containing Issue/PR filters do not work

### DISCUSSION THREAD

**Discussion by amadejpapez** [community/community#14500] [Mobile] [answered]
One example is Visual Studio Code releases: https://github.com/microsoft/vscode/releases/tag/1.66.1

They use `The update addresses these issues.` with a link that goes to their Issues page with Milestone and is:closed filters. Clicking on this inside of the app opens the Issue page without using set filters from the link.

I also tried using other links containing Issue/PR filters and it does not seem to work.

Is this supported by the app? Else it should open Safari as this can be confusing.

iOS 15.4.1, GitHub app 1.59.0

**Comment by iskae** [comment, marked as answer]
Hi @amadejpapez 👋 

Currently additional filters are not supported by our apps but I believe that this is a great idea 🎉  Thanks so much for your feedback. I have created an issue on our end to track it and hope to tackle it in the upcoming releases 👍

**Reply by amadejpapez** [reply]
It appears to be supported now! Thank you. Works amazingly 🚀
